extension.json

```diff
@@ -0,0 +1,58 @@
+{
+	"name": "SemanticExternalQueryLookup",
+	"version": "1.0.0-alpha",
+	"author": [
+		"James Hong Kong",
+		"Simon Heimler (gesinn.it GmbH & Co. KG)",
+		"Alexander Gesinn (gesinn.it GmbH & Co. KG)",
+		"Sebastian Schmid (gesinn.it GmbH & Co. KG)",
+		"Rcdeboer Remco C. de Boer"
```

Comment:
https://github.com/SemanticMediaWiki/SemanticExternalQueryLookup/graphs/contributors ?

Reply:
The gesinn.it team definitely did not contribute. Looks like a copy & paste.

Reply:
🤔 I have no idea from where this did arrive from, fixed